.github/workflows/publish_android.yml

```diff
@@ -3,6 +3,13 @@ name: Publish Android
 on:
   release:
     types: [published]
+  workflow_call:
+    inputs:
+      version:
+        required: false
+        type: string
+        description: 'The publication version'
+        default: '0.11.0-rc'
```

Comment:
It should be possible to add a job that parses this from `zenoh-kotlin/build.gradle.kts`